plugins/chruby/chruby.plugin.zsh

```diff
@@ -36,27 +43,14 @@ _source-from-homebrew() {
   source $_brew_prefix/share/chruby/auto.sh
 }
 
-_load-chruby-dirs() {
```

Comment:
FYI: This plugin's `_load-chruby-dirs` function is essentially doing the same thing `chruby`'s `share/chruby/chruby.sh` script [already does](https://github.com/postmodern/chruby/blob/a543a35790e5528b5a67de20e78a7390f5f7606e/share/chruby/chruby.sh#L5) for us, but the function is doing it wrong, i.e. `RUBIES+=("$dir")` instead of `RUBIES+=("$dir"/*)`, and that's the reason the output of `chruby` is broken, hence the removal of the function.